src/plots/cartesian/graph_interact.js

```diff
@@ -309,6 +309,13 @@ fx.unhover = function (gd, evt, subplot) {
 // The actual implementation is here:
 
 function hover(gd, evt, subplot){
+    if(gd._fullLayout._hasPie){
```

Comment:
Right, but `Fx.hover` for pie traces should emit that same [data](https://github.com/plotly/plotly.js/blob/master/src/plots/cartesian/graph_interact.js#L573) as for non-pie traces minus the info about the axes of course.